addons/website/static/src/builder/plugins/menu_data_plugin.js

```diff
@@ -44,8 +44,8 @@ export class MenuDataPlugin extends Plugin {
                     },
                     onClickEditMenu: () => {
                         this.services.dialog.add(EditMenuDialog, {
-                            save: () => {
-                                this.config.reloadEditor({ url: this.document.URL });
+                            save: async () => {
+                                await this.config.reloadEditor({ url: this.document.URL });
```

Comment:
Nitpicking: not really necessary as `EditMenuDialog` does not call the callback in an async context.